app/src/main/java/org/mozilla/fenix/home/MultiClickableSpan.kt

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.text.style.ClickableSpan
+import android.view.View
+
+object Position {
+    const val POS1 = 1
+    const val POS2 = 2
+    const val POS3 = 3
+}
+
+object ADDR {
+    const val URL1 = "https://www.mozilla.org/en-US/MPL/"
+    const val URL2 = "https://www.mozilla.org/en-US/foundation/trademarks/policy/"
+    const val URL3 = "https://www.mozilla.org/zh-CN/privacy/firefox/"
+}
+
+class MultiClickableSpan(var pos: Int, var context: Context) :
```

Comment:
This name implies it's a generic clickable span, but it isn't. How about naming it to be more specific? Maybe `PrivacyContentSpan`?

Or we could make this generic so it's re-usable for other places, but that might be out of scope for now.